pkg/agent/types/node_config_win.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Antrea Authors
```

Comment:
let's not include this in this PR

Reply:
this is required to compile windows antctl. will change the header though

Reply:
It seems that a better solution would be to refactor these type definitions and move them to another OS-agnostic package, instead of duplicating them. What is Windows-specific in this file?

Reply:
I agree, that would be idea. Perhaps another PR?

Reply:
node_config_win is removed after some code re-structure.